src/sentry/coreapi.py

```diff
@@ -174,16 +174,16 @@ def __init__(self, agent=None, version=None, project_id=None,
         self.log = ClientLogHelper(self.context)
 
     def auth_from_request(self, request):
-        if request.META.get('HTTP_X_SENTRY_AUTH', '').startswith('Sentry'):
+        if request.META.get('HTTP_X_SENTRY_AUTH', '')[:7] == 'Sentry ':
```

Comment:
This changes the check from `Sentry` to `Sentry `, notice the additional trailing space.

Inside `parse_auth_header`, we expect there to be at least 1 space in the header value which we split on, so it's reasonable to at least check for it here in case it were prefixed with `Sentryfoo` rather than `Sentry foo`, etc.

Also, these slice checks are faster than `.startswith` and we touch these pretty often.